diff --git a/debian-meta-data/freeplane.desktop b/debian-meta-data/freeplane.desktop
index c4bb83e8f..ed13e7761 100644
--- a/debian-meta-data/freeplane.desktop
+++ b/debian-meta-data/freeplane.desktop
@@ -1,13 +1,18 @@
 [Desktop Entry]
 Version=1.0
 Name=Freeplane
-Exec=/usr/bin/freeplane %F
+Exec=freeplane %F
 Terminal=false
-Icon=freeplane
+Icon=org.freeplane.App
 Type=Application
 MimeType=application/x-freeplane;
 Categories=Office;
-GenericName=Freeplane
-Comment=A free tool to structure and organise your information with mind mapping
-Keywords=Mindmaps; Knowledge management; Organize information; Brainstorming; ...;
+GenericName=Mind Maps
+GenericName[es]=mapas mentales
+Comment=Mind mapping, knowledge management and project management tool
+Comment[es]=Herramienta para mapas mentales, gestión del conocimiento y de proyectos
+
+Keywords=map;outline;draft;plan;project;brainstorming;
+Keywords[es]=mapa;esquema;borrador;plan;proyecto;tormenta de ideas;
+
 StartupWMClass=org-knopflerfish-framework-BundleThread
